### DISCUSSION TITLE
WooCommerce Add To Cart Buttons & Input Quantity

### DISCUSSION THREAD

**Discussion by digidoda** [bootscore/bootscore#624] [Q&A] [answered]
Hi all,

On a new install of bootscore, I've added a product options plugin, but it seems to break the formatting of the add to cart section whenever it is in use - do I need to edit any of the core files to allow the plugins to work correctly with the theme or should I reach out to the plugin developers as it doesn't seem to matter which options plugin I use, they all change the display of the add to cart section as per the attached screenshot:
![Screenshot 2023-11-13 at 15 59 38](https://github.com/bootscore/bootscore/assets/100483570/e7399a02-318e-4ac2-8828-cb1b961fdf94)

Many Thanks

Mike.

**Comment by crftwrk** [comment, marked as answer]
Please share a link to the product, because we need to debug this in the browser.

**Reply by digidoda** [reply]
Thanks for getting back to me so quickly - I've just copied it onto a dev server which you can access here:

https://wootesting-co-uk.stackstaging.com/product/sit-omnis/

Mike.

**Reply by crftwrk** [reply]
Please go in main-theme to

https://github.com/bootscore/bootscore/blob/65f923e688dd8c421d93ba5489489984f8b3274e/scss/bootscore_woocommerce/_wc_product_single.scss#L6-L8

and add `flex-wrap: wrap;` direct to line `9`. 

Let me know if that works. If so you can create a PR and we'll fix this in the theme.

**Reply by digidoda** [reply]
Hi Basti - that has sorted it all out now thank you.

**Reply by crftwrk** [reply]
Yes, do you want to create a pull request?

**Reply by crftwrk** [reply]
Quickly fixed it https://github.com/bootscore/bootscore/commit/2264033f10023f2dc9be6088c2fa03e8bca64217. 5.3.4 release is today.
